components/nexusoperations/tasks.go

```diff
@@ -198,7 +198,7 @@ type CancelationBackoffTask struct {
 	Deadline time.Time
 }
 
-var _ hsm.Task = CancelationBackoffTask{}
+var _ hsm.Task = (*CancelationBackoffTask)(nil)
```

Comment:
Why?

Reply:
No memory allocation? I usually see this way to check the type implements an interface.